app/javascript/components/change-logs/container.spec.js

```diff
@@ -0,0 +1,171 @@
+import { mountedComponent, screen } from "test-utils";
```

Comment:
Run eslint against this file. Use 2 spaces instead of 4

Reply:
done